bpf/bpf_host.c

```diff
@@ -249,6 +249,12 @@ handle_ipv6(struct __ctx_buff *ctx, __u32 secctx, const bool from_host)
 			return DROP_INVALID;
 	}
 
+#if !defined(ENABLE_NODEPORT) && defined(NO_REDIRECT) && !defined(ENABLE_REDIRECT_FAST)
+	/* See comment in handle_ipv4() */
+	if (!from_host)
+		return CTX_ACT_OK;
```

Comment:
What about the `skip_redirect` suggestion from https://github.com/cilium/cilium/pull/16136#discussion_r633339972 ?

Reply:
@jibi Does this comment need to be addressed?

Reply:
@brb to my understanding [the same conclusion for the v4 logic](https://github.com/cilium/cilium/pull/16136#discussion_r633477903) would apply here as well (i.e. following that suggestion we would skip redirect only if `ENABLE_NODEPORT` is not defined, ignoring the `NO_REDIRECT` macro, which is not what we want)